### PR TITLE
Spike on using Vultr

### PR DESCRIPTION
**Pull request by user512**
* TODO: Automatically fetch snapshot id to provision server
* TODO: Still need to figure out a way to get ssh_key_ids from terraform before packer build?

### CODE REVIEW

File: infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+variable "vultr_api_key" {
+  type = string
+}
+
+provider "vultr" {
+  api_key = var.vultr_api_key
+}
+
+resource "vultr_server" "convene_vultr_video" {
+  snapshot_id = "b765f07738a83"
```
Comment (zspencer) [inline infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf:42]: Could we parameterize this to a variable? Or can we use a [vultr_snapshot](https://www.terraform.io/docs/providers/vultr/d/snapshot.html) data source to filter to the `convene-jitsi-convene-test-vultr.zinc.coop*` description we use when building the image?

**Comment by zspencer** [comment]
Re: Keys - I wonder if we want to consider implementing #29 sooner rather than later; and using [Ansible's authorized key module](https://docs.ansible.com/ansible/latest/modules/authorized_key_module.html) to create users for each of us instead of relying on logging in as root/ubuntu.

Thoughts?

File: infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf (hunk 1 above)
Comment (user512) [reply infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf]: Couldn't figured out a way to do `convene-jitsi-convene-test-vultr.zinc.coop*` but parameterize Vultr Snapshot id.

Comment (zspencer) [reply infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf]: That's fine! This is great work!

**Comment by user512** [comment]
What do we gain by creating users instead of root?

**Comment by zspencer** [comment]
For one, it means we can both SSH into the same machine; for two it gives an audit log of who did what on the server when we sudo things.

There's a great thread here explaining the implications of signing in as root - https://askubuntu.com/questions/16178/why-is-it-bad-to-log-in-as-root

**Comment by user512** [comment]
Yes, that makes sense. The SSH into the same machine should be required before Andromeda release.

**Pull request merged**